### ISSUE TITLE
🐛 [BUG]: Uncaught TypeError: Cannot read properties of undefined (reading 'toLowerCase')

### ISSUE THREAD

**Issue by tpoxa** [bcakmakoglu/vue-flow#1241]
### Is there an existing issue for this?

- [X] I have searched the existing issues and this is a new bug.

### Current Behavior

I haven't updated for a while, so I did and noticed I have a bunch of errors when I select the node.
The come from this function 
```js
function isKeyMatch(pt, _t, xt, kt) {
    const Ct = _t.split("+").map(Et=>Et.trim().toLowerCase());
    return Ct.length === 1 ? pt.toLowerCase() === _t.toLowerCase() : (kt ? xt.delete(pt.toLowerCase()) : xt.add(pt.toLowerCase()),
    Ct.every((Et,$t)=>xt.has(Et) && Array.from(xt.values())[$t] === Ct[$t]))
}
```
### Expected Behavior

No errors

### Steps To Reproduce

It looks like this error is specific to my setup. If a nature of the error is completely not clear I'll try to provide some access to that page online. Is that going to help?

### Anything else?

<img width="1076" alt="Screenshot 2024-01-15 at 12 55 16" src="https://github.com/bcakmakoglu/vue-flow/assets/431546/b7e27563-5568-4982-8c44-60d0db207bf6">

**Comment by bcakmakoglu** [comment]
When does this issue happen exactly? Does it actually happen when you do some keypress combo?

**Comment by bcakmakoglu** [comment]
If you're using some keypress combo, what does it look like? Can you show me some of the code you use? :)

**Comment by tpoxa** [comment]
@bcakmakoglu just clicking on element, no keys pressed, that's very odd

**Comment by bcakmakoglu** [comment]
Selecting as in clicking a node or as in creating a selection-box? Just clicking a node doesn't seem to trigger the `isKeyMatch` function 🤔
Which version are you on right now?

Any more infos in the stacktrace? Like a hint where the error started from so that it ended up in the `isKeyMatch` section?

**Comment by bcakmakoglu** [comment]
Does this happen across all browsers or just specific ones?

**Comment by tpoxa** [comment]
ok, thanks for the tip, it happens only in Chrome. When I try incognito mode the error is not occurring even in chrome, so checking extensions

**Comment by bcakmakoglu** [comment]
Any updates on this? Were you able to find the issue?

**Comment by tpoxa** [comment]
> Any updates on this? Were you able to find the issue?

unfortunately I could not reproduce the issue, turning off extensions did not help

**Comment by bcakmakoglu** [comment]
Are you, by any chance, using any key-filters for any key-codes that might be empty arrays?

**Comment by bcakmakoglu** [comment]
I assume the problem is the same as #1255 

So this should be fixed in `1.29.2`

**Comment by bcakmakoglu** [comment]
Fixed in `1.29.2`

**Comment by tpoxa** [comment]
fixed indeed, thanks @bcakmakoglu
